venv_update.py

```diff
@@ -235,25 +234,13 @@ def get_original_path(venv_path):  # TODO-TEST: a unit test
     return check_output(('sh', '-c', '. %s; printf "$VIRTUAL_ENV"' % venv_executable(venv_path, 'activate')))
 
 
-def has_system_site_packages(interpreter):
-    # TODO: unit-test
-    system_site_packages = check_output((
-        interpreter,
-        '-c',
-        # stolen directly from virtualenv's site.py
-        """\
-import site, os.path
-print(
-    0
-    if os.path.exists(
-        os.path.join(os.path.dirname(site.__file__), 'no-global-site-packages.txt')
-    ) else
-    1
-)"""
-    ))
-    system_site_packages = int(system_site_packages)
-    assert system_site_packages in (0, 1)
-    return bool(system_site_packages)
+def has_system_site_packages(venv_path):
+    from pathlib import Path
+    from virtualenv.create.pyenv_cfg import PyEnvCfg
+    pyenv_cfg = PyEnvCfg.from_folder(Path(venv_path))
+    include_system_site_packages = pyenv_cfg['include-system-site-packages']
```

Comment:
I would recommend using `configparser` directly -- there's no benefits from using the `virtualenv` one here